test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightResourceSerializerTests.cs

```diff
@@ -619,147 +619,5 @@ private static ODataFunction CreateODataFunction(Uri resourceId, string function
         }
 
         #endregion Helper Methods
-

```

Comment:
I have provided an explanation for the changes in this test file in the PR description